test/regression.uts

```diff
@@ -6879,3 +6879,66 @@ x = f.i2repr(mp, {'*', '+', 'bit 2'})
 assert(re.match(r'^.*Star \(\*\).*$', x) is not None)
 assert(re.match(r'^.*Plus \(\+\).*$', x) is not None)
 assert(re.match(r'^.*bit 2.*$', x) is not None)
+
+###########################################################################################################
```

Comment:
The tests sound good to me, but I think you should call `inet_ntop` and let Scapy call either `socket.inet_ntop` or `_ipv6_bin_to_str()`, so that we really test how Scapy will behave. What do you think?